### PR TITLE
Integrate azure module code

### PR DESCRIPTION
**Pull request by Incarnation-p-lee**

### CODE REVIEW

File: src/main/java/com/microsoft/azure/springcloudplayground/service/ServiceNames.java [hunk 1]
```diff
@@ -67,4 +67,17 @@ public static Service toService(@NonNull MicroService microService) {
                         .build();
         }
     }
+
+    public static boolean isAzureService(@NonNull String serviceName) {
+        switch (serviceName) {
```
Comment (warrenzhu25) [inline src/main/java/com/microsoft/azure/springcloudplayground/service/ServiceNames.java]: I think azure service should start with `Azure`

File: src/main/java/com/microsoft/azure/springcloudplayground/generator/ProjectGenerator.java [hunk 2]
```diff
@@ -45,6 +47,19 @@
     @Setter
     private transient Map<String, List<File>> temporaryFiles = new LinkedHashMap<>();
 
+    private static final Map<String, List<String>> MODULE_TO_TEMPLATE;
```
Comment (warrenzhu25) [inline src/main/java/com/microsoft/azure/springcloudplayground/generator/ProjectGenerator.java]: What's the problem with original dynamic loading all templates under certain directory? I think the former way is more flexible

File: src/main/java/com/microsoft/azure/springcloudplayground/service/ServiceNames.java (hunk 1 above)
Comment (Incarnation-p-lee) [reply src/main/java/com/microsoft/azure/springcloudplayground/service/ServiceNames.java]: this name from user defined, and you cannot assure it start with `Azure`.

File: src/main/java/com/microsoft/azure/springcloudplayground/generator/ProjectGenerator.java (hunk 2 above)
Comment (Incarnation-p-lee) [reply src/main/java/com/microsoft/azure/springcloudplayground/generator/ProjectGenerator.java]: We can write another `write` directory for this.

Comment (sophiaso) [reply src/main/java/com/microsoft/azure/springcloudplayground/generator/ProjectGenerator.java]: Agree.

**Pull request merged**